### PR TITLE
tools/cmdautotest: Fix Syntax Python 2to3

### PR DESCRIPTION
**Pull request by KimDongEon**
Support for Python version 2 is coming to an end. Therefore, the syntax is modified to version 3.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6323a02314d8693e32dca1e295260b31ea258d98] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6323a02314d8693e32dca1e295260b31ea258d98] - Code Rule Check OK.

File: tools/cmdautotest/README_CMDAutoTest.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The CMD autotest automatically performs the tasks you need to perform when performing multiple tests on the TASH command and determines the results.
 
 ### Prerequisites
-Install python2.7, pyserial(at pip)
+Install python3, pyserial(at pip3)
```
Comment (chanijjani) [inline tools/cmdautotest/README_CMDAutoTest.md]: Does pip3 exist..?
As far as I know, "python3 -m pip install XXX" works.

File: tools/cmdautotest/README_CMDAutoTest.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 
 ### How to USE
 $ python cmdautotest.py serial_port
```
Comment (chanijjani) [inline tools/cmdautotest/README_CMDAutoTest.md]: python or python3?

File: tools/cmdautotest/README_CMDAutoTest.md (hunk 1 above)
Comment (KimDongEon) [reply tools/cmdautotest/README_CMDAutoTest.md]: The python2 version uses pip, but python3 uses pip3.

File: tools/cmdautotest/README_CMDAutoTest.md (hunk 2 above)
Comment (KimDongEon) [reply tools/cmdautotest/README_CMDAutoTest.md]: Yes, it is exactly python3.x version, but abbreviated as python3.

Comment (KimDongEon) [reply tools/cmdautotest/README_CMDAutoTest.md]: Support for Python version 2 is coming to an end. Therefore, the syntax is modified to version 3.

**Pull request merged**